external_parser/joiners/example_joiner.cc

```diff
@@ -420,11 +420,23 @@ bool example_joiner::process_joined(v_array<example *> &examples) {
     }
   }
 
+  joined_event::joined_event *je = nullptr;
```

Comment:
making this a pointer so that it can be available before the scope guard and also available if we need it during cleanup